lib/aws_codegen/docstring.ex

```diff
@@ -78,44 +257,42 @@ defmodule AWS.CodeGen.Docstring do
   def justify_line(text, max_length \\ 75, indent \\ "  ") do
     text
     |> break_line(max_length)
-    |> Enum.map(&(String.trim(&1)))
-    |> Enum.reject(&(&1 == ""))
-    |> Enum.map(&("#{indent}#{&1}"))
+    |> Enum.map(&String.trim_trailing(&1))
+    |> Enum.map(fn line ->
+      if line == "" do
+        line
+      else
+        "#{indent}#{line}"
+      end
+    end)
     |> Enum.join("\n")
   end
 
-  defp convert_links(text) do
-    Regex.replace(~r{<a href="(.+?)">(.+?)</a>}, text, "[\\2](\\1)",
-                  global: true)
-  end
-
   defp break_line(text, max_length) do
-    {lines, current} = List.foldl(String.split(text), {[], ""},
-      fn word, {lines, current} ->
-        case String.length(current) + 1 + String.length(word) > max_length do
-          true ->
-            case current == "" do
-              true ->
-                # The current word is the first on the current line, and is
-                # longer than our max_length, so append it as a new line.
-                {lines ++ [word], current}
-              false ->
-                # The current word exceeds the max length, so append the last
-                # line, and start a new one with the current word.
-                {lines ++ [current], word}
-            end
-          false ->
-            case current == "" do
-              true ->
-                # The current word is the first on the current line, so append
-                # it as a new line.
-                {lines, word}
-              false ->
-                # Append the current word to the current line.
-                {lines, "#{current} #{word}"}
-            end
+    {lines, current} =
```

Comment:
I guess we can improve this function using IO data. But it' s OK for now.

Reply:
Agreed, it would be faster but not a big concern here. :)